### ISSUE TITLE
Font properties dialog

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#192]
**Reported by tvalusek on 17 Jun 2008 06:35 UTC**
There are some windows in NVDA displaying text. It would be great to add an option to set font and background color properties using standard Windows dialogs.

**Comment by nvdaTrac** [comment]
**Comment by Bernd on 8 Nov 2008 17:14 UTC**
Does NVDA need such a feature? I think you can change the font in windows by it self. Does NVDA use the windows fonts?

**Comment by nvdaTrac** [comment]
**Comment by mdcurran on 10 May 2010 07:21 UTC**
I don't personally think that NVDA needs this feature. If there is a major issue with the font that NVDA uses, then a new ticket should be opened describing the specific issue, and it can be fixed if possible.
Also closing due to inactivity.
